cmd/function-sidecar.go

```diff
@@ -32,37 +32,71 @@ import (
 	"github.com/sk8sio/function-sidecar/pkg/dispatcher"
 	"github.com/sk8sio/function-sidecar/pkg/message"
 	"github.com/sk8sio/function-sidecar/pkg/dispatcher/grpc"
+	"flag"
```

Comment:
nitpicky, but it's nice to have std lib imports all together at the top (I think Goland automatically adds to the bottom, at least by default)